core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -96,19 +93,15 @@ angular.module('oppia').directive('aboutPage', [
               ' & ' + listOfNamesToThank[listOfNamesToThank.length - 1];
             ctrl.aboutPageMascotImgUrl = UrlInterpolationService
               .getStaticImageUrl('/general/about_page_mascot.png');
-            ctrl.listOfNames = listOfNamesToThank
```

Comment:
Why deleted, here and below?

If you delete anything in non-test files, please add an explanation as a comment, otherwise the reviewer is just going to ask for one and it delays things.

Reply:
`listOfNames` and `aboutPageMascotImgUrl` is duplicated in the line above.